modules/core/perf/opencl/perf_arithm.cpp

```diff
@@ -1038,14 +1039,40 @@ OCL_PERF_TEST_P(ConvertScaleAbsFixture, ConvertScaleAbs,
 
 ///////////// PatchNaNs ////////////////////////
 
+template<typename _Tp>
+_Tp randomNan(int seed);
+
+template<>
+float randomNan(int seed)
+{
+    uint32_t r = RNG(seed).next();
+    Cv32suf v;
+    v.u = r;
+    // exp & set a bit to avoid zero mantissa
+    v.u = v.u | 0x7f800001;
+    return v.f;
+}
+
+template<>
+double randomNan(int seed)
+{
+    uint32_t r0 = RNG(seed).next();
+    uint32_t r1 = RNG(seed).next();
```

Comment:
You got 2 same numbers.

Don't use `seed`. Use `cv::theRNG()`

Reply:
Since `theRNG()` depends on the order in which tests are being run, will use an `RNG(0)` to get deterministic sanity test data

Reply:
> Since theRNG() depends on the order in which tests are being run

Example?
This should be fixed for the last 5+ years.